tests/unit/s2n_x509_validator_test.c

```diff
@@ -1041,9 +1036,8 @@ int main(int argc, char **argv) {
         EXPECT_SUCCESS(read_file(&ocsp_data_stuffer, S2N_OCSP_RESPONSE_DER, S2N_MAX_TEST_PEM_SIZE));
         uint32_t ocsp_data_len = s2n_stuffer_data_available(&ocsp_data_stuffer);
         EXPECT_TRUE(ocsp_data_len > 0);
-        EXPECT_EQUAL(S2N_CERT_ERR_EXPIRED, s2n_x509_validator_validate_cert_stapled_ocsp_response(&validator, connection,
-                                                                                                  s2n_stuffer_raw_read(&ocsp_data_stuffer, ocsp_data_len),
-                                                                                                  ocsp_data_len));
+        EXPECT_ERROR_WITH_ERRNO(s2n_x509_validator_validate_cert_stapled_ocsp_response(&validator, connection,
+                s2n_stuffer_raw_read(&ocsp_data_stuffer, ocsp_data_len), ocsp_data_len), S2N_ERR_CERT_INVALID);
```

Comment:
This is a behavior change. Why isn't "S2N_CERT_ERR_EXPIRED" correct here?

Reply:
Currently an expired error is used both for when the current time is > nextUpdate, but also when the current time is < thisUpdate. The latter didn't really seem like an expiration to me, so I changed it to invalid. Though, would it make more sense to just use the expired error if there's a problem with the thisUpdate/nextUpdate timestamp fields?

Reply:
Yeah that's a tough one. I think you're right, and S2N_ERR_CERT_INVALID is more correct.